webbpsf/gridded_library.py

```diff
@@ -421,3 +430,63 @@ def writeto(self, data, meta, detector):
             print("  Saving file: {}".format(file))
 
         hdu.writeto(file, overwrite=self.overwrite)
+
+
+def display_psf_grid(grid, zoom_in=True):
+    """ Display a PSF grid in a pair of lpots
+
+    Shows the NxN grid in NxN subplots, repeated to show
+    first the individual PSFs, and then their differences
+    from the average PSF.
+
+    At this time, this only visualizes a single GriddedPSFModel object,
+    i.e.  covering one detector, not an entire instrument field of view.
+
+    This function returns nothing, but makes some plots.
+
+    Inputs
+    -------
+    grid : photutils.GriddedPSFModel object
+        The grid of PSFs to be displayed.
+
+
+    """
+    import matplotlib
+    import matplotlib.pyplot as plt
+
+    tuple_to_int = lambda t: (int(t[0]), int(t[1]))
+    def show_grid_helper(grid, data, title="Grid of PSFs", vmax=0, vmin=0, scale='log'):
+        npsfs = grid.data.shape[0]
+        n = int(np.sqrt(npsfs))
+
+        fig, axes = plt.subplots(n, n, figsize=(12,12))
```

Comment:
```suggestion
        fig, axes = plt.subplots(n, n, figsize=(12, 12))
```

Reply:
I went with a slightly different fix on this one, making the `figsize` into a function parameter so it can be changes at runtime if desired.